src/DssSpell.sol

```diff
@@ -59,187 +51,224 @@ contract DssSpellAction is DssAction {
+        // Forum: https://forum.sky.money/t/risk-assessment-and-parameter-recommendations-spark-ddm-to-aave-lido-market/25175
 
-        // ---------- Spark Spell ----------
-        // Forum: https://forum.makerdao.com/t/sep-12-2024-proposed-changes-to-spark-for-upcoming-spell/25076
-        // Poll: https://vote.makerdao.com/polling/QmPFkXna
 
-        // Execute Spark Proxy Spell at 0xc80621140bEe6A105C180Ae7cb0a084c2409C738
-        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
+        // Set D3M DC-IAM with the following parameters:
+        // line: 100 million USDS
+        // gap: 50 million USDS
+        // ttl: 24 hours
+        // tau: 7 days
+        // D3M Addresses:
+        // oracle: D3MOracle
+        // plan: D3MOperatorPlan
+        // pool: D3MAaveV3USDSNoSupplyCapTypePool
+        // aToken: TBC
+        // operator: 0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5
+        // stabledebt address: TBC
+        // variabledebt address: TBC
+        // Additional Actions
+        // Expand DIRECT_MOM breaker to also include new D3M
+        // Note: this is already done within D3MInit.sol line 232
+        // D3MInstance memory d3m = D3MInstance({
+        //     plan:   DIRECT_SPK_AAVE_LIDO_USDS_PLAN,
+        //     pool:   DIRECT_SPK_AAVE_LIDO_USDS_POOL,
+        //     oracle: DIRECT_SPK_AAVE_LIDO_USDS_ORACLE
+        // });
+        // D3MCommonConfig memory d3mCfg = D3MCommonConfig({
+        //     hub:         D3M_HUB,
+        //     mom:         D3M_MOM,
+        //     ilk:         "DIRECT-SPK-AAVE-LIDO-USDS",
+        //     existingIlk: false,
+        //     maxLine:     100 * MILLION * RAD,
+        //     gap:         50 * MILLION * RAD,
+        //     ttl:         24 hours,
+        //     tau:         7 days
+        // });
+        // D3MAaveUSDSPoolConfig memory aaveCfg = D3MAaveUSDSPoolConfig({
+        //     king:         address(0), // TODO: replace with TBC
+        //     ausds:        address(0), // TODO: replace with TBC
+        //     usdsJoin:     address(0), // TODO: replace with TBC
+        //     usds:         address(0), // TODO: replace with TBC
+        //     stableDebt:   address(0), // TODO: replace with TBC
+        //     variableDebt: address(0) // TODO: replace with TBC
+        // });
+        // D3MOperatorPlanConfig memory operatorCfg = D3MOperatorPlanConfig({
+        //     operator: DIRECT_SPK_AAVE_LIDO_USDS_OPERATOR
+        // });
+        // D3MInit.initCommon({
+        //     dss:     dss,
+        //     d3m:     d3m,
+        //     cfg:     d3mCfg
+        // });
+        // D3MInit.initAaveUSDSPool({
+        //     dss:     dss,
+        //     d3m:     d3m,
+        //     cfg:     d3mCfg,
+        //     aaveCfg: aaveCfg
+        // });
+        // D3MInit.initOperatorPlan({
+        //     d3m: d3m,
+        //     operatorCfg: operatorCfg
+        // });
+        // ---------- Update WBTC Legacy Vaults Parameters  ----------
```

Comment:
Can we add both [link1](https://forum.makerdao.com/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/48) and [link2](https://forum.sky.money/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/52) as they lead to different part of the forum page?

Reply:
same as above